project/src/components/reviews-form/reviews-form.tsx

```diff
@@ -0,0 +1,38 @@
+import {useState} from 'react';
+import Rating from '../rating/rating';
+
+type FieldsType = {
+  target: {
+    name: string;
+    value: string;
+  }
+}
+
+function ReviewsForm(): JSX.Element {
+  const [formData, setFormData] = useState({
+    rating: '',
+    review: '',
+  });
+
+  const fieldChangeHandle = (evt: FieldsType) => {
+    const {name, value} = evt.target;
+
+    setFormData({...formData, [name]: value});
+  };
+
+  return (
+    <form className="reviews__form form" action="#" method="post">
+      <label className="reviews__label form__label" htmlFor="review">Your review</label>
+      <Rating onRatingChange={fieldChangeHandle} />
+      <textarea onChange={fieldChangeHandle} className="reviews__textarea form__textarea" id="review" name="review" value={formData.review} placeholder="Tell how was your stay, what you like and what can be improved"></textarea>
```

Comment:
Я бы пустыми строками для наглядности бы отделил
```suggestion
      <label className="reviews__label form__label" htmlFor="review">Your review</label>
      <Rating onRatingChange={fieldChangeHandle} />
      
      <textarea onChange={fieldChangeHandle} className="reviews__textarea form__textarea" id="review" name="review" value={formData.review} placeholder="Tell how was your stay, what you like and what can be improved"></textarea>
```